### ISSUE TITLE
Fix a11y: Inputs do not have a visible focus state when HC Light theme is applied

### ISSUE THREAD

**Issue by hawkticehurst** [microsoft/vscode#160225]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: `Stable 1.71.0` / `Insiders 1.72.0-insider`
- OS Version: `macOS 12.5.1`

Steps to Reproduce:

1. Change editor to use the VS Code High Contrast default light theme
2. Open the settings page
3. Find a setting that uses a checkbox
4. Tab to focus on the checkbox (you should see no visible change when focused)

**Reference/Comparison Screenshots**

Default Dark Theme:
<img width="276" alt="Screen Shot 2022-09-06 at 1 33 53 PM" src="https://user-images.githubusercontent.com/39639992/188733904-a9310ecd-d363-4d56-b170-c017cf5210ec.png">
<img width="288" alt="Screen Shot 2022-09-06 at 1 33 58 PM" src="https://user-images.githubusercontent.com/39639992/188733908-96c2a218-aa3b-46b1-b95b-a1e7c7984454.png">

Default Light Theme:
<img width="275" alt="Screen Shot 2022-09-06 at 1 34 16 PM" src="https://user-images.githubusercontent.com/39639992/188733975-6b445340-5e31-41e2-a380-cd0479a13f31.png">
<img width="278" alt="Screen Shot 2022-09-06 at 1 34 22 PM" src="https://user-images.githubusercontent.com/39639992/188733977-ab90840c-d4bc-4033-8abe-808ddef2410a.png">

Default HC Dark Theme:
<img width="281" alt="Screen Shot 2022-09-06 at 1 33 29 PM" src="https://user-images.githubusercontent.com/39639992/188734032-c84c8a5b-0d31-4fb4-8a9e-8897ee543bba.png">
<img width="278" alt="Screen Shot 2022-09-06 at 1 33 37 PM" src="https://user-images.githubusercontent.com/39639992/188734035-a34f1dad-175d-4713-890b-1c4dfd0d8239.png">

Default HC Light Theme (note lack of difference between non-focused/focused states):
<img width="280" alt="Screen Shot 2022-09-06 at 1 34 51 PM" src="https://user-images.githubusercontent.com/39639992/188734102-28ecc1d5-ba34-4198-914c-87010dad2581.png">
<img width="274" alt="Screen Shot 2022-09-06 at 1 34 58 PM" src="https://user-images.githubusercontent.com/39639992/188734103-792f3d0b-0a86-43c4-b08e-890a4efcd9b4.png">

cc @daviddossett

**Comment by hawkticehurst** [comment]
Oh, looks like this issue seems to apply to dropdown and text field elements as well. I can file separate issues for each UI element if that's helpful/desired.

**Comment by rzhao271** [comment]
There's a difference now, but it's very slight, imo. I'm wondering if another colour like orange could be used instead for the focus border, like in the HC dark theme?
![Focused setting next to an unfocused setting with the HC light theme](https://user-images.githubusercontent.com/7199958/214702217-19de09df-864e-4e14-aa13-e3210e3f64e4.png)
